python/python_fastapi_server/routers/edr.py

```diff
@@ -420,11 +471,16 @@ def get_params_for_collection(edr_collection: str) -> dict[str, Parameter]:
         else:
             label = param_el["name"]
 
+        symbol = Symbol(value=param_el["unit"], type=SYMBOL_TYPE_URL)
+        param_id = param_el["name"]
+        # If standard name was configured, use that instead with a vocabulary
+        if "standard_name" in param_el:
+            param_id = VOCAB_ENDPOINT_URL + param_el["standard_name"]
         param = Parameter(
             id=param_el["name"],
-            observedProperty=ObservedProperty(id=param_el["name"], label=label),
+            observedProperty=ObservedProperty(id=param_id, label=label),
```

Comment:
We are using the same `label` in the `ObservedProperty` and the `Parameter` technically this is not correct. The property is an "air temperature", the parameter is "air temperature measured at 2 m altitude with sensor X".

I think the label provided in the example config is appropriate for the `parameter`, but not for the `ObservedProperty`.

Reply:
Can we use the standard_name (`air_temperature`) in the label of `observedProperty` instead, or should we support an additional configuration option?

Probably an additional config option?